### ISSUE TITLE
Is there a way to programmatically switch to next or previous page?

### ISSUE THREAD

**Issue by casey55** [fortmarek/SwipeViewController#33]
Is there a way to programmatically switch to next or previous page?

**Comment by fortmarek** [comment]
'SwipeViewController is a subclass of 'UIPageViewController', so it should work the same. Let me know how it goes.

**Comment by casey55** [comment]
Wanted to try that be doing something like the following, but don't have access to the underlying "pageController" 

`pageController.setViewControllers([myViewController], direction: .forward, animated: true, completion: nil)
`
I thought SwipeViewController was subclass of UINavigationController that contains a UIPageViewController and access to the underlying "pageController" is not allowed.

**Comment by fortmarek** [comment]
You're right, I'm on a vacation now, so I'd suggest to either just open the underlying pageController, and I'll later look into it, or, which I would obviously prefer, create a PR where you would create a function to turn to the next page programmatically while keeping the pageController internal to the SwipeViewController.

**Comment by casey55** [comment]
Will do.  Thanks!
